src/services/categoryApi.ts

```diff
@@ -0,0 +1,58 @@
+import axiosInstance from "~/api/axiosInstance";
+import { Category } from "~/types/Category";
+
+export interface CreateCategoryRequest {
+	name: string;
+	description?: string;
+}
+
+export interface UpdateCategoryRequest {
+	name?: string;
+	description?: string;
+}
+
+export interface CategoryResponse {
+	data: Category;
+	message: string;
+}
+
+export interface SearchCategoryResponse {
+	data: Category[];
+	message: string;
+}
+
+// Add category
+export const addCategoryApi = (
```

Comment:
Tương tự các file khác
```suggestion
export const add = (
```